src/components/common/ListResolutions.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div
+    v-if="resolutionList.length > 0"
+    id="list-resolutions"
+  >
+    <v-divider class="mx-4" />
+    <article class="pa-4">
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4"
+        >
+          <label
+            id="resolution-label"
+            class="font-weight-bold"
+          >Resolution or court Order Dates</label>
+        </v-col>
+        <v-col
+          cols="12"
+          sm="9"
+          class="pt-4 pt-sm-0"
+        >
+          <p>
+            Dates of resolution or court orders to alter the company's share structure or the
+            special rights or restrictions to a class or a series of shares:
+          </p>
+        </v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4"
+        />
+        <v-col
+          cols="12"
+          sm="9"
+          class="pt-4 pt-sm-0"
```

Comment:
On a narrow screen, pt-4 is too much.

![image](https://github.com/user-attachments/assets/cc5820c9-7263-4c3d-a40c-5c447364b038)

I think you can use pt-0 or maybe it's not even needed.